docs/notebooks/regional_frequency_analysis.ipynb

```diff
@@ -0,0 +1,970 @@
+{
```

Comment:
<p>Cette phrase n'est pas claire. Est-ce que ça serait plutôt:</p><p> </p><blockquote>We need to reshape our datasets of annual maximums and L-moments according to the groupings found using the clustering algorithm. Since there is no convention on the name of that new dimension, it has been decided in xHydro that it would need to be called <code>group_id</code>.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hydrologie/xhydro/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>